src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -920,9 +926,9 @@ namespace winrt::TerminalApp::implementation
     //   than one tab opened, show a warning dialog.
     void TerminalPage::CloseWindow()
     {
-        if (_tabs.size() > 1)
+        if (_tabs.size() > 1 && _settings->GlobalSettings().GetConfirmCloseAllTabs())
         {
-            _ShowCloseWarningDialog();
+            _ShowCloseWarningDialog();            
```

Comment:
These trailing spaces are probably why the code formatting check failed 😉

Reply:
Ah OK, I was wondering about that. I'm used to VSCode where trailing whitespace is cleaned up for me automatically.